clients/fossid-webapp/src/test/kotlin/FossIdClientNewProjectTest.kt

```diff
@@ -107,8 +101,8 @@ class FossIdClientNewProjectTest : StringSpec({
 
     "Scans for project can be listed when there is no scan" {
         service.listScansForProject("", "", PROJECT_CODE) shouldNotBeNull {
-            checkResponse("list scans")
-            toList<Scan>() should beEmpty()
+            checkResponse("list scans", true)
+            data!!.values should beEmpty()
```

Comment:
It is a bit ugly to have these "!!" everywhere. Could this be circumvented somehow, maybe by having another helper function `checkNonNullResponse()`, which returns a non-nullable object?

Reply:
Or maybe just be a bit more verbose and use kotest functions, e.g.
```
data shouldNotBeNull { values should beEmpty() }
```
`with(data!!)` and `data!!.should` could be replaced with this, and `with(data!!.values)` could be replaced with `data shouldNotBeNull { with(data.vlues) { ... } }`.

Reply:
Fixed thanks for the hint.
Just for the record, these `!!` were not unsafe because `data` was never `null`: the previous call in `checkResponse` made sure of that.
However, using `shouldNotBeNull` as suggested makes it more straightforward to the reader.